### PR TITLE
Deprecations and index page review

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-graphql-7.surge.sh

File: modules/ROOT/pages/getting-started/toolbox.adoc [hunk 1]
```diff
@@ -2,10 +2,16 @@
 
 = Neo4j GraphQL Toolbox
 
-Experiment with your Neo4j GraphQL API using the purpose-built web IDE.
+Neo4j GraphQL Toolbox is an onboarding, embeddable low-code tool that allows writing and executing GraphQL queries and mutations against your Neo4j database, with absolutely no setup required. Intrigued to see more? Let’s try it out together!
```
Comment (darrellwarde) [inline modules/ROOT/pages/getting-started/toolbox.adoc:5]: I'm not sure I would use the word "embeddable" here!

```suggestion
Neo4j GraphQL Toolbox is an onboarding, low-code tool that allows writing and executing GraphQL queries and mutations against your Neo4j database, with absolutely no setup required. Intrigued to see more? Let’s try it out together!
```

File: modules/ROOT/pages/getting-started/toolbox.adoc [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
 
 > This tool is for development and experimentation purposes only!
 
+we released a beta version of a developer user interface to build a powerful GraphQL API using the @neo4j/graphql library, which we named the Neo4j GraphQL Toolbox.
+
+We also made it publicly available under https://graphql-toolbox.neo4j.io. I wrote an article back when we released the beta version, which offers an introduction to the web application Neo4j GraphQL Toolbox. A very brief intro is further available on our documentation page.
```
Comment (darrellwarde) [inline modules/ROOT/pages/getting-started/toolbox.adoc:11]: Are these meant to be here? 🧐

File: modules/ROOT/pages/deprecations.adoc [hunk 3]
```diff
@@ -1,60 +1,30 @@
...
-== What has happened to the GraphQL Architect?
+== Neo4j GraphQL Architect
 
-From a technical perspective we have deprecated and removed the GraphQL Architect Graph App from npm and the https://install.graphapp.io/[Graph App gallery], it is no longer available to be installed in Neo4j Desktop.
-The associated GitHub repositories have been archived.
+Neo4j GraphQL Architect was a low-code graph app for Neo4j Desktop that enabled developers to build, query, and deploy GraphQL APIs backed by the Neo4j graph database.
+It was deprecated and removed from npm, the https://install.graphapp.io/[Graph App gallery], and Neo4j Desktop.
+The associated GitHub repositories have also been archived.
 
+This tool was repurposed and expanded with xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox], an onboarding, embeddable low-code tool that can be integrated to Neo4j.
```
Comment (darrellwarde) [inline modules/ROOT/pages/deprecations.adoc:30]: This makes it sound like it was a rewrite, so I would maybe choose different wording here. Also remove "embeddable" again.

```suggestion
This tool was reimagined in the xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox], an onboarding, low-code tool that can be integrated to Neo4j.
```

File: modules/ROOT/pages/deprecations.adoc [hunk 4]
```diff
@@ -1,60 +1,30 @@
...
-
-It was also attractive to junior developers with very little experience in building applications, and it was a great channel to bring a new audience to Neo4j. 
+- xref:deprecations.adoc#_grandstack_starter_app[GRANDstack starter app]
+- xref:deprecations.adoc#_neo4j_graphql_architect[Neo4j GraphQL Architect]
 
-Finally, it was helpful for users who had an existing front end and needed a new backend. They needed a way to initiate a server, so the value we offered was by providing an opinionated mechanism that can help users get started with Neo4j with GraphQL. 
+== GRANDstack starter app
 
-Over time, the GRANDstack starter grew to support other frameworks such as Flutter and Angular. With hindsight, this massively increased the support burden and technical debt of the product, and it is time to revisit the scope of the product.
+The main purpose of the GRANDstack starter app to demonstrate how the Neo4j Labs GraphQL library could be used in the context of a full-stack application using React and Apollo client.
```
Comment (darrellwarde) [inline modules/ROOT/pages/deprecations.adoc:11]:
```suggestion
The main purpose of the GRANDstack starter app was to demonstrate how the Neo4j Labs GraphQL library could be used in the context of a full-stack application using React and Apollo client.
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,81 +1,75 @@
...
 
-The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform and mobile applications by tapping into the power of connected data.
+[NOTE]
+====
+The GRANDstack starter app has been deprecated. 
+For more information, read the section on xref::deprecations.adoc[Deprecations].
+====
 
-With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have application data treated as a graph natively from the front-end all the way to storage, avoiding duplicate schema work and ensuring flawless integration between front-end and backend developers.
+The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform, and mobile applications by tapping into the power of connected data.
```
Comment (darrellwarde) [inline modules/ROOT/pages/index.adoc:11]:
```suggestion
The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform and mobile applications by tapping into the power of connected data.
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -1,81 +1,75 @@
...
 
-Written in TypeScript, the library's schema-first paradigm lets developers focus on the application data they need, while taking care of the heavy lifting involved in building the API.
+With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have data treated as a graph natively from the front-end all the way to storage.
+This avoids duplicate schema work and ensures flawless integration between front-end and backend developers.
 
-TIP: Are you a GRANDstack user? We have deprecated the GRANDstack starter app, find more information xref::deprecations.adoc[here].
+== How it works
 
-== How does it work?
+Written in TypeScript, the Neo4j GraphQL Library offers a set of type definitions that describes the shape of your graph data.
```
Comment (darrellwarde) [inline modules/ROOT/pages/index.adoc:18]: I would remove the language, and highlight that the library requires type definitions of the user to generate the schema.

```suggestion
The Neo4j GraphQL Library requires a set of type definitions that describes the shape of your graph data.
```

**Review by darrellwarde — COMMENTED** [review]
Looks mostly good as far as I can see! 😄

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**